Wrapping/Generators/Python/Tests/extras.py

```diff
@@ -409,6 +409,16 @@ def custom_callback(name, progress):
     output_image = input_image.astype(output_pixel_type)
     assert type(output_image) == itk.Image[output_pixel_type, 2]
 
+if '(<itkCType bool>, 4)' in itk.Image.GetTypesAsList():
+    arr = np.random.randint(0, 255, size=(3,4,5,6), dtype=np.uint8)
```

Comment:
I assume that `np.uint8` is the storage type for `<itkCType bool>`.  If not, is this right?

Reply:
Good catch. Addressed.